tests/zlib-simple/source/data/config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "parameters": {
+    "compression_type": "zlib"
```

Comment:
prihodil bych i test na window_size parametr (at je to jednou s a jednou bez, at se testuje i ten default)

Reply:
přidán